docs/guide/custom-snipptes.md

```diff
@@ -1,7 +1,12 @@
 # Custom Snippets
-Assumed that you need a custom function for your elements, hvstr has the custom snippets feature.
-It allows you to add code to the page-object generation process, which will be added,
-if a given condition is true.
+Assumed that you need hvstr to generate some custom functionality for your elements. Therefore hvstr has the custom snippets feature.
```

Comment:
Therefore heißt "deswegen" -> Kausalität :)

Muss mich damit aber gleich entschuldigen.

Ich kannte nur "therefore" und nicht "therefor" und habe es dir einmal verbessert.

Habe es eben extra gegooglet und "therefor" heißt "dafür" also hast du es korrekt verwendet.
Scheint aber "juristische" Sprache zu sein.

Gibt also therefore und therefor.

Finde "For that" oder "For this" gebräuchlicher aber werde dir "therefor" nicht mehr verbwessern.